packages/quantic/cypress/page-objects/search.ts

```diff
@@ -0,0 +1,37 @@
+export const InterceptAliases = {
+  UA: {
+    Facet: {
+      ClearAll: '@coveoUaFacetClearAll',
+      Search: '@coveoUaFacetSearch',
+      Select: '@coveoUaFacetSelect',
+    },
+  },
+  QuerySuggestions: '@coveoQuerySuggest',
+  Search: '@coveoSearch',
+  FacetSearch: '@coveoFacetSearch',
+};
+
+export const interceptSearch = () =>
+  cy
+    .intercept('POST', '**/rest/ua/v15/analytics/*', (req) => {
+      switch (req.body.actionCause) {
+        case 'facetClearAll':
```

Comment:
@lvu Having aliases on specific UA requests helped made it easier in the test because I did not need to "skip" UA requests like `interfaceLoad` to get to the right request.

The only case where I had to "skip" request was to validate that a second facet value is selected.